packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -72,6 +73,70 @@ export class CapabilityService {
     );
   }
 
+  /**
+   * Handle a Stripe Webhook subscription change.
+   *
+   * This handles broadcasting and refreshing the subscription capabilities for
+   * the product ids that were possibly updated.
+   *
+   * Stripe supports aligned subscriptions such that a single subscription can
+   * include multiple items for multiple products. We iterate through this list
```

Comment:
afaik we do not use the multiple product per subscription feature.  Do we plan to in the near future?

Reply:
We're not using it at the moment, but its easy enough to support now rather than making things more difficult in the future. We have had aligned subscriptions requested in the past.